codebuild/common-posix.sh

```diff
@@ -3,42 +3,45 @@
 set -e
 
 CMAKE_ARGS="$@"
+BUILD_PATH="/tmp/builds"
+mkdir -p $BUILD_PATH
+INSTALL_PATH="$BUILD_PATH/install"
+mkdir -p $INSTALL_PATH
 
 function install_library {
+    pushd
+    cd $BUILD_PATH
```

Comment:
fun fact, you can do pushd $BUILD_PATH to save yourself a line

Reply:
:+1: